dockerfiles/Dockerfile.heroku-24

```diff
@@ -0,0 +1,14 @@
+FROM heroku/heroku:24-build
+
+USER root
+
+RUN apt-get update -y; apt-get install default-jre default-jdk -y
```

Comment:
Technically we don't need these for downloading and re-packaging jruby, but they're needed to be able to execute jruby to verify that the packaged tar file works as expected.